image.go

```diff
@@ -13,18 +13,38 @@ const (
 	CreateImageSize256x256   = "256x256"
 	CreateImageSize512x512   = "512x512"
 	CreateImageSize1024x1024 = "1024x1024"
+	// dall-e-3 supported only.
+	CreateImageSize1792x1024 = "1792x1024"
+	CreateImageSize1024x1792 = "1024x1792"
 )
 
 const (
 	CreateImageResponseFormatURL     = "url"
 	CreateImageResponseFormatB64JSON = "b64_json"
 )
 
+const (
+	CreateImageModelDallE2 = "dall-e-2"
+	CreateImageModelDallE3 = "dall-e-3"
+)
+
+const (
+	CreateImageQualityHD = "hd"
```

Comment:
@mikecutalo 
Sometimes you may want to explicitly specify default values in the code. Please add the following as well.

`CreateImageQualityStandard = "standard"`

<img width="321" alt="image" src="https://github.com/sashabaranov/go-openai/assets/9883589/3fcf06ac-e975-4f87-b390-5e27b94c9289">

https://platform.openai.com/docs/api-reference/images/create#images-create-quality